rllib/execution/rollout_ops.py

```diff
@@ -32,16 +32,16 @@ def synchronous_parallel_sample(
     If no remote workers exist (num_workers == 0), use the local worker
     for sampling.
 
-    Alternatively to calling `worker.sample.remote()`, the user can provide a
-    `remote_fn()`, which will be applied to the worker(s) instead.
+    if neither max_agent_steps or max_env_steps are specified, then this method will 
+    only call `sample()` once on each worker, and will not check if the number of agent 
+    or env steps are met. If one of these is not None, then this method will call 
+    `sample()` on each worker until the number of agent or env steps is met.
 
     Args:
         worker_set: The WorkerSet to use for sampling.
-        remote_fn: If provided, use `worker.apply.remote(remote_fn)` instead
-            of `worker.sample.remote()` to generate the requests.
-        max_agent_steps: Optional number of agent steps to be included in the
+        max_agent_steps: The minimum number of agent steps to be included in the
             final batch.
-        max_env_steps: Optional number of environment steps to be included in the
+        max_env_steps: The minimum number of environment steps to be included in the
```

Comment:
I don't think these names are intuitive in any way. They are actually the minimum_env_steps and not max_env_steps. I don't want to change them in this PR, since it will induce massive change on a lot of files. For not keep it this way.